lib/redacted_struct.rb

```diff
@@ -1,47 +1,41 @@
 # frozen_string_literal: true
 
-# A subclass of Struct that redacts members by default, and can allow some to be printed
-class RedactedStruct < Struct
-  VERSION = "1.1.0"
-
-  def self.new(*name_and_members, keyword_init: nil, allowed_members: [], &block)
-    super(*name_and_members, keyword_init: keyword_init, &block).tap do |struct_class|
-      struct_class.class_eval do
-        @allowed_members = Array(allowed_members)
-      end
-    end
-  end
-
-  class << self
-    attr_reader :allowed_members
-  end
+# Class methods for the Redactable module. When Redactable is included in class that class is
+# extended with these instance methods.
+module RedactableClassMethods
+  attr_reader :allowed_members
+end
 
-  def allowed_members
-    self.class.allowed_members
+# A module that adds redacted functionality to a class
+module Redactable
```

Comment:
I think it might be easier to find things if we broke these into separate files?

- `redacted_struct.rb`
- `redactable.rb`
- `redacted_data.rb`

and this one (`redacted_struct.rb`) would automatically include the other two

Reply:
I'm up for that! Willfix